pye3d/geometry/primitives.py

```diff
@@ -110,6 +120,8 @@ def __bool__(self):
 
 
 class Conicoid(Primitive):
+    __slots__ = tuple("ABCFGHUVWD")
```

Comment:
Does this class miss an `E` attribute by any chance? Could we also add links to a mathematical reference by any chance?